### ISSUE TITLE
WebStorm Support

### ISSUE THREAD

**Issue by Caesar2011** [justjavac/typescript-deno-plugin#34]
I tried to use this plugin in WebStorm 2020.1 . But I still getting the following error. I installed "typescript" and "typescript-deno-plugin" to my dev dependencies. Also I generated a tsconfig.json with the content below. In VS Code everything works as expected and no error is shown with the `axetroy.vscode-deno` deno plugin installed. Thank you for your work!

Error Message of WebStorm:

    Error:(1, 29) TS2691: An import path cannot end with a '.ts' extension. Consider importing 'https://raw.githubusercontent.com/NMathar/deno-express/master/mod' instead.

src/mod.ts

```
import * as expressive from "https://raw.githubusercontent.com/NMathar/deno-express/master/mod.ts";

const port = 3000;
const app = new expressive.App();
app.use(expressive.simpleLog());
app.use(expressive.static_("./public"));
app.use(expressive.bodyParser.json());
app.get("/api/todos", async (req, res) => {
  await res.json([{ name: "Buy some milk" }]);
});
// route with dynamic parameter
app.get("/api/user/{user_id}", async (req, res) => {
  await res.json([{ id: req.params.user_id, name: "Jim Doe", phone: "12425323" }]);
});
const server = await app.listen(port);

console.log("app listening on port " + server.port);
```

tsconfig.json

```
{
  "compilerOptions": {
    "module": "commonjs",
    "target": "es5",
    "sourceMap": true,
    "plugins": [
      {
        "name": "typescript-deno-plugin"
      }
    ]
  },
  "include": [
    "src/mod.ts"
  ],
  "exclude": [
    "node_modules"
  ]
}
```

**Comment by Caesar2011** [comment]
Also I have no `.vscode` folder in my project.

**Comment by justjavac** [comment]
Yes, this is a known bug and I'm fixing it

**Comment by justjavac** [comment]
fixed by c345c395e2ce5b3d79a1c0a93f5466b3c082f060
